1.8/administration/access-node/tunnel.md

```diff
@@ -0,0 +1,113 @@
+---
+post_title: Access by Proxy
+menu_order: 10
+---
+
+When developing applications on DC/OS, you may find it helpful to access your cluster via SOCKS proxy, HTTP proxy, or VPN. For instance, you can work from your own development environment and immediately test on your DC/OS cluster.
+
+# SOCKS
+dcos tunnel can run a SOCKS proxy over SSH to the cluster. SOCKS proxies work for any kind of network protocol on any port, but your application must be configured to use the proxy, which runs on port 1080 by default. Access URLs by appending the `mydcos.directory` domain.
+
+# HTTP
+Run the HTTP proxy on your local port 80 to tunnel HTTP requests without modifying your application. Access URLs by appending the `mydcos.directory` domain. You can also [use DNS SRV records as if they were URLs](#srv).
+
+<a name="srv"></a>
+## SRV Records
+An SRV DNS record is a mapping from a name to a IP/port pair. DC/OS creates SRV records in the form `_<port-name>._<task-name>._tcp.marathon.mesos.mydcos.directory`. The HTTP proxy exposes these as URLs. This feature can be useful for communicating with Marathon apps.
```

Comment:
Maybe instead of "Marathon apps" we should call them "DC/OS Services" (I think that's what they're called, may want to verify that with someone else as well).